README.md

```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
 
 ## What we are about
 
-We have several primary objectives with this linter:
+We have several primary objectives with regards to this linter:
 
-0. Enforce `python3.6+` usage
-1. Significantly reduce code's complexity and make it more maintainable
-2. Enforce "There should be one-- and preferably only one --obvious way to do it" rule
+0. Enforce usage Python 3.6 or higher
```

Comment:
Can you please revert this specific line?

Reply:
Okay, will do :)